### PR TITLE
Updated list of dependencies for Reveal SDK

### PR DESCRIPTION
**Pull request by federicoprovera**

### CODE REVIEW

File: docs/web/third-party-software.md [hunk 1]
```diff
@@ -1,18 +1,72 @@
...
-| Web          | [Babel JS Compiler](https://github.com/babel/babel)                                                           | [MIT](https://opensource.org/licenses/mit-license.html)                                                    |
-| Web          | [Playwright for .NET](https://github.com/microsoft/playwright-dotnet)                                         | [MIT](https://opensource.org/licenses/mit-license.html)                                                    |
-| Web          | [Simple JavaScript Inheritance](https://johnresig.com/blog/simple-javascript-inheritance/)                    | [MIT](https://opensource.org/licenses/mit-license.html)                                                    |
\ No newline at end of file
+# Third-Party Software Used by Reveal SDK (web)
+
+|Platform|Dependency|License|
+|--------|----------|-------|
+|ASP .NET|[Newtonsoft.Json](https://www.newtonsoft.com/json)|[MIT](https://opensource.org/licenses/mit-license.html)|
```
Comment (brianlagunas) [inline docs/web/third-party-software.md:5]: why add a column for the platform when you can create a sub heading in markdown?

## ASP.NET
...

## Node.js
...

## Java
...

**Comment by brianlagunas** [comment]
when you update the web md files to use headers instead of an additional column for the platform, be sure to copy/paste the information into the japanese docs too 
image

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: docs/web/third-party-software.md [hunk 2]
```diff
@@ -1,72 +1,74 @@
...
-|ASP .NET|[System.IO.Packaging](https://dot.net/)|[MIT](https://opensource.org/license/mit/)|
-|ASP .NET|[System.Runtime.Caching](https://dotnet.microsoft.com/en-us/)|[MIT](https://opensource.org/license/mit/)|
-|ASP .NET|[Microsoft.AspNetCore.NodeServies](https://asp.net/)|[Apache 2.0](https://opensource.org/license/apache-2-0/)|
-|ASP .NET|[System.Drawing.Common](https://github.com/dotnet/runtime)|[MIT](https://opensource.org/license/mit/)|
-|ASP .NET|[DocumentFormat.OpenXML](https://github.com/dotnet/Open-XML-SDK)|[MIT](https://opensource.org/license/mit/)|
-|ASP .NET|[AngleSharp](https://github.com/AngleSharp/AngleSharp)|[MIT](https://opensource.org/license/mit/)|
-|ASP .NET|[System.Text.RegularExpresions](https://dot.net/)|[MS .NET Library](https://dotnet.microsoft.com/en-us/dotnet_library_license.htm)|
-|ASP .NET|[log4net](https://logging.apache.org/log4net/)|[Apache 2.0](https://opensource.org/license/apache-2-0/)|
-|ASP .NET|[System.Configuration.ConfigurationManager](https://dot.net/)|[MIT](https://opensource.org/license/mit/)|
+**ASP .NET**
```
Comment (brianlagunas) [inline docs/web/third-party-software.md:3]: please use markdown heading with a double "#" for example: `## ASP.NET`

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: docs/web/third-party-software.md (hunk 2 above)
Comment (federicoprovera) [reply docs/web/third-party-software.md]: Done.

**Review by brianlagunas — APPROVED** [review]

**Pull request merged**